app/policies/order_policy.rb

```diff
@@ -0,0 +1,12 @@
+class OrderPolicy
+  attr_reader :user, :order
+
+  def initialize(user, order)
+    @user = user
+    @order = order
+  end
```

Comment:
If you want, you can create a `BasePolicy` class that other classes inherit from so we don't have to redefine `initialize` every time, e.g.:

https://github.com/18F/identity-dashboard/blob/8b06d08356cd0bf21558613168001987c5d4e0a4/app/policies/base_policy.rb#L1-L7

Reply:
👍  wasn't sure what defaults would be good, I ran `rails g pundit:install` which sets up the basic defaults and am now inheriting from that policy